### PR TITLE
Reduce test-clients images proposal

### PR DESCRIPTION
**Pull request by im-konge**
Signed-off-by: Lukas Kral <lukywill16@gmail.com>

This PR adds proposal for reducing number of images built for each test-clients release.

### CODE REVIEW

File: 039-reduce-test-clients-images.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+* `quay.io/strimzi-test-clients/test-client-http-{client-type}:{version}`
+
+will be changed to `quay.io/strimzi-test-clients/test-clients:{version}-kafka-{kafka-version}`.
+
+### `Systemtest` implementation and changes 
+
+* Environment variables will be reduced
+  * `TEST_PRODUCER_IMAGE`, `TEST_CONSUMER_IMAGE` and other environment variables will be removed
+  * `TEST_CLIENT_IMAGE` will be added
+* Container configuration of each client will contain `args` section with appropriate `run.sh`
```
Comment (Frawless) [inline 039-reduce-test-clients-images.md:50]:
```suggestion
* Container configuration of each client will contain `args` section with appropriate `*_run.sh`
```

**Review by Frawless — COMMENTED** [review]
LGTM, maybe you can add that older versions of systemtest will use already released clients so there should be no issue with compatibility. Maybe mentioning the size of the images would be nice as well (current and new one)

**Comment by im-konge** [comment]
@Frawless I will surely add the size of the image - once I create some prototype. Currently it's just guessing.

**Review by Frawless — APPROVED** [review]

**Review by ppatierno — APPROVED** [review]

**Comment by scholzj** [comment]
@im-konge I guess this can be merged now as approved? You seem to have 4 +1 votes plus your own? Just don't forget to update the index before merging it.

**Comment by im-konge** [comment]
@scholzj  before merging this I wanted to create some prototype or something. But if you agree on merging this, I can update the index and work on the PR to the test-clients repository.

**Comment by scholzj** [comment]
Well, I think this is fairly straight forward. You can of course create the prototype first if you want. But I'm not sure what issues you expect.

**Pull request merged**